mappings/net/minecraft/client/search/IdentifierSearcher.mapping

```diff
@@ -7,5 +7,5 @@ CLASS net/minecraft/unmapped/C_mkylkzob net/minecraft/client/search/IdentifierSe
 		ARG 1 path
 	METHOD m_mbaochwd empty ()Lnet/minecraft/unmapped/C_mkylkzob;
 	METHOD m_nzgsulfj create (Ljava/util/List;Ljava/util/function/Function;)Lnet/minecraft/unmapped/C_mkylkzob;
-		ARG 0 list
+		ARG 0 searchables
 		ARG 1 identifiers
```

Comment:
```suggestion
		ARG 1 ids
```
`ids` is a [conventional abbreviation](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#abbreviations) for `identifiers`